osu/objects/news.py

```diff
@@ -70,3 +75,6 @@ class Navigation:
     def __init__(self, data):
         self.newer = NewsPost(data['newer']) if data["newer"] is not None else None
         self.older = NewsPost(data['older']) if data["older"] is not None else None
+
+    def __repr__(self):
```

Comment:
Is there a reason for leaving the fields blank here?

Reply:
not really, solved in 844861766b61ccffad332d19c808cb9a558e82d2